packages/react-composites/src/composites/CallComposite/pages/CallPage.tsx

```diff
@@ -89,9 +95,28 @@ export const CallPage = (props: CallPageProps): JSX.Element => {
   const errorBarProps = usePropsFor(ErrorBar);
   const mutedNotificationProps = useSelector(mutedNotificationSelector);
   const networkReconnectTileProps = useSelector(networkReconnectTileSelector);
+  /* @conditional-compile-remove(dtmf-dialer) */
+  const remoteParticipantsConnected = useSelector(getRemoteParticipantsConnectedSelector);
+
+  /* @conditional-compile-remove(dtmf-dialer) */
+  const adapter = useAdapter();
+  /* @conditional-compile-remove(dtmf-dialer) */
+  const callees = adapter.getState().targetCallees;
+  /* @conditional-compile-remove(dtmf-dialer) */
+  const renderDtmfDialerFromStart = showDtmfDialer(callees);
+
+  /* @conditional-compile-remove(dtmf-dialer) */
+  const [dtmfDialerPresent, setDtmfDialerPresent] = useState<boolean>(renderDtmfDialerFromStart);
+  /* @conditional-compile-remove(dtmf-dialer) */
+  const [initialDialerDismissed, setInitialDialerDismissed] = useState<boolean>(false);
 
   /* @conditional-compile-remove(dtmf-dialer) */
-  const [dtmfDialerPresent, setDtmfDialerPresent] = useState<boolean>(false);
+  useEffect(() => {
```

Comment:
This useeffect is not obvious on its purpose, add comment around this being used to dismiss this dialer if a second participant joins